packages/animation-curve-block/src/components/BlankSlate.tsx

```diff
@@ -0,0 +1,98 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEffect, useState } from 'react';
+
+import { IconPlus24, generateRandomId } from '@frontify/fondue';
+
+import { AnimationCurve, AnimationCurveType, BlankSlateProps, defaultAnimationCurveTypeValues } from '../types';
+import { AnimationCurveFlyout, CardText } from './';
+
+export const BlankSlate = ({
+    appBridge,
+    content,
+    designTokens,
+    hasBorder,
+    lineColor,
+    endpointColor,
+    gridColor,
+    canvasHeight,
+    setLocalItems,
+    setBlockSettings,
+}: BlankSlateProps) => {
+    const height = canvasHeight !== 0 ? `${canvasHeight}px` : 'auto';
+    const [title, setTitle] = useState('');
+    const [description, setDescription] = useState('');
+    const [isFlyoutOpen, setIsFlyoutOpen] = useState(false);
+    const defaultAnimationCurve: AnimationCurve = {
+        id: generateRandomId(),
+        title: '',
+        description: '',
+        animationFunction: {
+            type: AnimationCurveType.EaseInOut,
+            parameters: defaultAnimationCurveTypeValues[AnimationCurveType.EaseInOut],
+            duration: 1.0,
+        },
+    };
+
+    const [animationCurve, setAnimationCurve] = useState<AnimationCurve>(defaultAnimationCurve);
+    const addAnimationCurve = async (id: string) => {
+        setLocalItems([...(content ?? []), { ...animationCurve, id }]);
+        await setBlockSettings({
+            content: [...(content ?? []), { ...animationCurve, id: animationCurve.id }],
+        });
+        setIsFlyoutOpen(false);
+        setTitle('');
+        setDescription('');
+    };
+
+    useEffect(() => {
+        setAnimationCurve({ ...animationCurve, title });
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [title]);
+
+    useEffect(() => {
+        setAnimationCurve({ ...animationCurve, description });
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [description]);
+
+    return (
+        <div data-test-id="blank-slate">
+            <AnimationCurveFlyout
+                animationCurve={animationCurve}
+                onSave={addAnimationCurve}
+                onCancel={() => setIsFlyoutOpen(false)}
+                onOpenChange={(isOpen) => setIsFlyoutOpen(isOpen)}
+                onAnimationCurveChange={setAnimationCurve}
+                isFlyoutOpen={isFlyoutOpen}
+                lineColor={lineColor}
+                endpointColor={endpointColor}
+                gridColor={gridColor}
+            />
+            <button
```

Comment:
Can't use it because of the height.

There is a problem with the layout i didn't mention, within figma the slate button and the animation curve is always 200px height, but this won't work as you are stretching the curve and so you are also not displaying the "real" curve, so I decided to it a bit different and display the slate always in the height of the other animation curves so it looks great.

Reply:
Yep that sounds good. But BlockInjectButton has a prop called `fillParentContainer` so you can customise its dimensions.

Reply:
yep, replaced it with the BlockInjectButton